src/pages/Doeun/Main/Components/Nav/Nav.js

```diff
@@ -0,0 +1,73 @@
+import React from 'react';
+import './Nav.scss';
+import { Link, withRouter } from 'react-router-dom';
+
+class Nav extends React.Component {
+  constructor() {
+    super();
+    this.state = {
+      matchedUsers: [],
+    };
+  }
+
+  goToLogin = () => {
+    this.props.history.push('/login-doeun');
+  };
+
+  search = e => {
+    this.setState({
+      matchedUsers: this.props.users.filter((user, index) =>
+        user.id.startsWith(e.target.value)
+      ),
+    });
+  };
+
+  render() {
+    const { matchedUsers } = this.state;
+
+    return (
+      <nav>
+        <div>
+          <h1 onClick={this.goToLogin}>westagram</h1>
+          <form className="search">
+            <input
+              type="text"
+              placeholder="검색"
+              onClick={this.search}
+              onInput={this.search}
+            />
+            <i className="fas fa-search"></i>
+            <div className="searchList">
+              <div className="idList">
+                {!matchedUsers.length ? (
+                  <p>검색결과가 없습니다.</p>
+                ) : (
+                  matchedUsers.map((users, index) => (
+                    <div key={index} className="searchedUser">
+                      <img alt="user's profile image" src={users.profileImg} />
```

Comment:
img alt 속성도 templete literal 을 사용해서 좀더 정확하게 표현해주면 좋을 것 같아요 ! 
```suggestion
                      <img alt={`${users.id}'s profile image`} src={users.profileImg} />
```

Reply:
일일히 줘야한다고 생각햇는데 그런 좋은 방법이...... 짱감사해요👍👍🧡